src/controllers/handlers.ts

```diff
@@ -148,6 +148,16 @@ export const estateMapPngRequestHandler = async (context: {
     const selected = Object.values(tiles).filter(
       (tile) => tile.estateId && tile.estateId === estateId
     )
+    if (selected.length === 0) {
+      const  headers: Record<string, string> = {
+        'content-type': 'image/png',
```

Comment:
Is the content-type necessary here? when the redirection occurs, the `https://ui.decentraland.org/dissolved_estate.png` should respond with the content type of the image.